local-config/fabric/README.md

```diff
@@ -0,0 +1,145 @@
+# fabric-client
+fabric-client is a Node.js Hyperledger Fabric blockchain client that allows transactions to be sent to the blockchain via a REST API or a NATS listener.  The fabric-client works with the fhir-data smart contract to provide storage of FHIR patient resources in the blockchain.
+
+## Pre-requisites
+If you don't have a Hyperledger Fabric instance and would like to install the Hyperledger Fabric test-network on your local machine, you can follow the "testing with test-network" instructions below.
+
+## Install the contract
+This repo contains fhir-data.tar.gz which is a Hyperledger Fabric Typescript contract for storing FHIR-R4 Patient records in the blockchain.  Install this contract in your Hyperledger Fabric network, or follow the instructions below to install it in a test-network instance.
+
+## Test with test-network
+The fabric client can be tested using a local Hyperledger Fabric test-network instance.  Follow the steps below to set up test-network on your local machine.
+
+### Install Hyperledger fabric-samples
+Follow the [instructions](https://hyperledger-fabric.readthedocs.io/en/latest/getting_started.html) to install the Hyperledger Fabric samples repository and pre-requisites.
+
+### Start the test-network
+```shell
+cd fabric-samples/test-network
+./network.sh up createChannel -c channel1 -ca
+```
+
+### Copy the contract to the test-network
+Copy the contract to the test-network directory.  It will be installed in later step.
+```shell
+cp <fabric-client-path>/fabric-client/fhir-data.tar.gz .
+```
+
+### Install the contract
+
+#### Install the contract (chaincode) as Org1.
```

Comment:
I will likely create a script that does the whole contract deployment workflow, but it may be in a subsequent PR.